src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -200,16 +200,21 @@ public PersistentDataStore() {
 
     return activities;
   }
-  public List<Setting> loadSetting()throws PersistentDataStoreException{
+
+  public List<Setting> loadSettings()throws PersistentDataStoreException{
     List<Setting> setting = new ArrayList<>();
     Query query = new Query("settings");
     PreparedQuery results = datastore.prepare(query);
 
     for(Entity entity : results.asIterable()){
```

Comment:
this shouldn't be a for loop since our list of settings is coming from pulling different properties from settings and not from iterating.

Reply:
got it.